OpenRA.Mods.Common/Widgets/Logic/ConnectionLogic.cs

```diff
@@ -142,4 +142,4 @@ public ConnectionFailedLogic(Widget widget, OrderManager orderManager, Action on
 			}
 		}
 	}
-}
\ No newline at end of file
+}
```

Comment:
note: it always pays to click on the "Files changed" tab above and check that you've only committed things that are directly related to the pr.